generate.sh

```diff
@@ -17,6 +17,10 @@ docker run --rm -v "${PWD}:/local" -u $(id -u) openapitools/openapi-generator-cl
 mv README.md client.md
 mv README.md.bak README.md
 
+# improve time format from second to nanosec
+sed -i .bak s/\(time.RFC3339\)/\(time.RFC3339Nano\)/g ./client.go
```

Comment:
Ubuntuだと以下のエラーが出ました
`-i .bak`を空白挟まずにくっつけるとよさそうです
```
+ sed -i .bak s/(time.RFC3339)/(time.RFC3339Nano)/g ./client.go
sed: -e expression #1, char 1: unknown command: `.'
```

```suggestion
sed -i.bak s/\(time.RFC3339\)/\(time.RFC3339Nano\)/g ./client.go
```